rpc/src/main/scala/org/bitcoins/rpc/config/BitcoindInstance.scala

```diff
@@ -23,6 +25,21 @@ sealed trait BitcoindInstance {
   def zmqConfig: ZmqConfig
 
   def rpcPort: Int = authCredentials.rpcPort
+
+  def getVersion: BitcoindVersion = {
```

Comment:
See other comment about potentially problematic IO operation

Reply:
What is the main concern here? Is this going to be getting called a lot?

Reply:
It's a IO operation (although a very modest one) that's done by blocking. It's called on startup (to check we have the correct version), and when using utility methods such as `signRawTransaction` where we create a versioned RPC client from an unversioned one